cionic/dsp.py

```diff
@@ -395,19 +435,25 @@ def processRawData(time_raw_us, data_bits, v_ref, channel_gain):
     print('---DSP: PROCESSRAWDATA---')
 
     # Calculate Time array (starts at 0)
-    #time_sec = np.arange(0, len(data_bits)*(1/self.sampling_freq ), 1/self.sampling_freq) # units: seconds (s)
-    time_sec = (time_raw_us -  time_raw_us[0])/ 1000000.
+    time_sec = (time_raw_us - time_raw_us[0]) / 1000000.0
 
     # Convert data from bits to voltage
-    data_uV = 1e6 * data_bits * v_ref / (channel_gain * (2**23 - 1)) # units: microvolts (uV)
+    data_uV = (
+        1e6 * data_bits * v_ref / (channel_gain * (2**23 - 1))
+    )  # units: microvolts (uV)
 
     # Return time (sec) and data (uV)
-    return time_sec, data_uV  #these are pandas Series (RMS and FFT data are numpy arrays). Should probably pick one format and convert all others to it.
+    # these are pandas Series (RMS and FFT data are numpy arrays).
+    # Should probably pick one format and convert all others to it.
+    return (
+        time_sec,
+        data_uV,
+    )
 
 
 def setGain(which_channel, gainlevel, isRLD):
     """
-    Calculate channel gain by accounting for gainlevel (i.e. gain set in the INIT.f file)
+    Calculate channel gain by accounting for gainlevel (i.e. gain set in the INIT.f)
```

Comment:
nit (though not sure if it matters in the fw): I've always labeled my files `init.f` - don't know if the fw will complain if it's all caps